src/main/java/com/poc/kubeappswrapper/service/KubeAppsOrchitestratorService.java

```diff
@@ -131,6 +137,21 @@ private void proceessTrigger(CustomerDetails customerDetails, String targetNames
 
 			Map<String, String> map = dftWorkFlow.getWorkFlow(customerDetails, action, inputConfiguration,
 					createTrigger);
+
+			//Send an email
+			Map<String, Object> emailContent = new HashMap<>();
+			emailContent.put("name", customerDetails.getOrganizationName());
+			emailContent.put("dftfrontendurl", map.get("dftfrontendurl"));
+			emailContent.put("dftbackendurl", map.get("dftbackendurl"));
+			EmailRequest emailRequest = EmailRequest.builder()
+					.emailContent(emailContent)
+					.subject("DFT Application Deployed Successfully")
+					.templateFileName("success.html")
+					.build();
+			emailManager.sendEmail(emailRequest);
```

Comment:
Move this logic to email manager. Pass customerDetails and map requireddetails to emailmanager let manager has to prepare that email object and perform email send process